sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.1.10 (Unreleased)
+## 1.2.0 (2020-10-16)
```

Comment:
I don't think we should push through a change to a complicated piece of deserialization code on a Friday. Can we make this Monday at the earliest?

Reply:
Done